src/screens/LoginForm/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { t } from 'i18next';
+import { Link, useHistory } from 'react-router-dom';
+import clsx from 'clsx';
+
+import logo from '../../app/assets/logo.png';
+import ButtonForm from '../../components/ButtonForm';
+import InputText from '../../components/InputText';
+import { emailPattern } from '../../utils/formValidations';
+import { login } from '../../services/userService';
+import useLazyRequest from '../../app/hooks/useLazyRequest';
+import { AUTH_INPUTS } from '../../constants/forms';
+import { PATHS } from '../../constants/paths';
+
+import styles from './styles.module.scss';
+
+function LoginForm() {
+  const history = useHistory();
+  const { register, handleSubmit, errors } = useForm();
+  const [isLoading, submitError, response, sendRequest] = useLazyRequest({ request: login });
+  const onSubmit = handleSubmit(({ email, password }) => {
+    sendRequest({
+      email,
+      password
+    });
+  });
+
+  if (response?.ok) {
+    history.push(PATHS.root);
+  }
+
+  return (
+    <div className="column center">
+      <div className={clsx(styles.loginForm, 'column')}>
```

Comment:
use clsx only to validate properties. 
```suggestion
      <div className={`column ${styles.loginForm}`}>
```

Reply:
It's fine either way in my opinion